agent/command/timeout.go

```diff
@@ -66,8 +66,13 @@ func (c *timeout) Execute(ctx context.Context, _ client.Communicator, logger cli
 		logger.Execution().Infof("Set idle timeout to %d seconds.", c.TimeoutSecs)
 	}
 	if c.ExecTimeoutSecs != 0 {
-		conf.SetExecTimeout(c.ExecTimeoutSecs)
-		logger.Execution().Infof("Set exec timeout to %d seconds.", c.ExecTimeoutSecs)
+		if c.ExecTimeoutSecs > conf.ExecTimeoutLimit {
+			conf.SetExecTimeout(conf.ExecTimeoutLimit)
+			logger.Execution().Errorf("Exec timeout %d seconds exceeds the limit of %d seconds. Set exec timeout to %d seconds.", c.ExecTimeoutSecs, conf.ExecTimeoutLimit, conf.ExecTimeoutLimit)
```

Comment:
Should this continue on error, or return on error? Seems like erroring out early might be preferable, since the user would expect it to be set to whatever they choose.

If we're going to continue on error, I would suggest putting this log in the task log rather than execution logs because it's relevant to this command and the execution logs are usually for background operations that aren't in the user's control, not for task command logs.

Reply:
I thought that we should continue on error because it technically doesn't affect any run time behavior and I don't think users typically aim to hit the timeout 

but I will log in task instead of execution!